core/domain/config_domain.py

```diff
@@ -224,7 +226,6 @@ def update_moderator_ids():
     [])
 
 
-# TODO(milit): Check the language codes are valid
 LANGUAGES_SUPPORTED = ConfigProperty(
```

Comment:
As described in a previous comment, I still think this should be a constant in feconf instead -- I'm not sure it's a good idea to support arbitrary easy adding of languages by the admin, especially if the translations aren't present. I'd also recommend adding the backend test here that checks that the contents of this constant correspond to valid language codes, in order to prevent against skew in the future.